From 2a61d014d2a82862144bb4b6b655fba9368efb31 Mon Sep 17 00:00:00 2001
From: Etaash Mathamsetty
 <45927311+Etaash-mathamsetty@users.noreply.github.com>
Date: Sat, 18 Jul 2026 02:15:57 -0400
Subject: [PATCH 205/239] winewayland: Avoid deadlocks with
 update_client_surfaces.

Just straight up remove it because it just deadlocks no matter where it is.
---
 dlls/winewayland.drv/wayland_surface.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/dlls/winewayland.drv/wayland_surface.c b/dlls/winewayland.drv/wayland_surface.c
index 53afb279d9d..ec6c0b2d7cf 100644
--- a/dlls/winewayland.drv/wayland_surface.c
+++ b/dlls/winewayland.drv/wayland_surface.c
@@ -238,9 +238,6 @@ void wp_fractional_scale_handle_scale(void* user_data,
 
     surface->window.scale = scale;
 
-    /* reattach client surfaces as their rects have changed */
-    update_client_surfaces(hwnd);
-
     wayland_win_data_release(data);
 
     NtUserExposeWindowSurface(hwnd, 0, NULL, 0);
-- 
2.55.0

